skyportal/facility_apis/sedmv2.py

```diff
@@ -118,6 +118,11 @@ def submit(request):
         DBSession().add(transaction)
 
         flow = Flow()
+        flow.push(
+            '*',
+            'skyportal/REFRESH_SOURCE',
+            payload={'obj_key': request.obj.internal_key},
+        )
         flow.push(
             '*',
             "skyportal/REFRESH_FOLLOWUP_REQUESTS",
```

Comment:
These pushes trigger:

```
messageHandler.add((actionType, payload, dispatch) => {
  if (actionType === REFRESH_FOLLOWUP_REQUESTS) {
    dispatch(fetchFollowupRequests());
  }
});
```

Isn't that what we're trying to avoid?

Reply:
Ah yes. Lemme remove those.